### PR TITLE
chore(advertising): update copies

### PR DESCRIPTION
**Pull request by caugner**
## Summary

Updates the copy texts of the advertising pages now that the experiment has been finished.

### Changes

1. Removed the "Experimenting with advertising on MDN" page.
2. Updated the "Advertise with us" page.
3. Moved the "Advertise with us" page to `/advertising`.
4. Added a redirect from `/advertising/with_us` to `/advertising`.

---

## Screenshots

| Path | Before | After |
|---|---|---|
| `/advertising` | image | image
| `/advertising/with_us` | image | (Removed.)

---

## How did you test this change?

Ran `yarn dev` locally and visited both http://localhost:3000/advertising and http://localhost:3000/advertising/with_us.

### CODE REVIEW

**Review by fiji-flo — APPROVED** [review]
Please add and redirect and fix all links (`rg advertising` should help). Otherwise great!

File: client/src/advertising/with_us/index.tsx [hunk 1]
```diff
@@ -51,46 +51,89 @@ export function AdvertiseWithUs() {
...
-          extensive set of <a href="/en-US/docs/Learn">learning resources</a>{" "}
-          for new developers and students, accounting for approximately 10% of
-          our audience. More than 17 million web developers visit our website
-          every month, and we pride ourselves with a customer satisfaction of
-          over 90%.
+          At MDN, we champion user privacy and we stand by{" "}
+          <a
+            href="https://www.mozilla.org/en-US/privacy/ad-targeting-guidelines/"
+            target="_blank"
+            rel="noreferrer"
```
Comment (caugner) [inline client/src/advertising/with_us/index.tsx:60]:
```suggestion
```

File: client/src/advertising/with_us/index.tsx [hunk 2]
```diff
@@ -51,46 +51,89 @@ export function AdvertiseWithUs() {
...
             className="external"
           >
-            here
+            Mozilla's Ad targeting guidelines
+          </a>{" "}
+          and{" "}
+          <a
+            href="https://www.mozilla.org/en-US/privacy/"
+            target="_blank"
+            rel="noreferrer"
```
Comment (caugner) [inline client/src/advertising/with_us/index.tsx:133]:
```suggestion
```

**Review by caugner — COMMENTED** [review]
@fiji-flo Maybe keep the referrer for Mozilla-internal links?

File: client/src/advertising/with_us/index.tsx (hunk 1 above)
Comment (fiji-flo) [reply client/src/advertising/with_us/index.tsx:60]: No strong pref here. Mainly a habit.

Comment (caugner) [reply client/src/advertising/with_us/index.tsx:60]: That way bedrock colleagues would see it in their stats.

Comment (caugner) [reply client/src/advertising/with_us/index.tsx:60]: Never mind, ESLint adds it back, I reverted my change using force-push.

**Pull request merged**